### DISCUSSION TITLE
Bot .dig function flagging on anticheats

### DISCUSSION THREAD

**Discussion by Drownek** [PrismarineJS/mineflayer#3298] [Q&A]
The [FAQ](https://github.com/PrismarineJS/mineflayer/blob/master/docs/FAQ.md) doesn't contain a resolution to my issue

Versions
mineflayer: 4.18.0

Hello, im trying to make a bot that will mine "stone generator" that bot is facing. But when i run it, i see that the bot is flagging on anticheats that i installed on my server (Grim and Vulcan). It is flagging only for packet alerts, like this:
![image](https://github.com/PrismarineJS/mineflayer/assets/46686155/e5178f2e-43b3-4ed1-b3df-28dd5817b78a)
![image](https://github.com/PrismarineJS/mineflayer/assets/46686155/261684df-349f-40d1-8e14-4b6208899188)

I have tried many things from doing a loop in a different way to delaying next .dig functions, but it is flagging no matter what i try. I think its something with packet order in mineflayer or something wrong i'm doing.

I would be very grateful for your help because I have been looking for a solution for several weeks now

here is my current code:
```js
    const bot = mineflayer.createBot({
        host: "0.0.0.0", // optional
        port: 25565, // optional
        version: 1.8,
        auth: "microsoft"
    });
```
```js
    bot.on("physicsTick", () => {
        const target = bot.blockAtCursor(4.5);
        if (!bot.targetDigBlock) {
            if (target && bot.canDigBlock(target)) {
                bot.dig(target, 'ignore').catch()
            }
        }
    });
```

**Comment by extremeheat** [comment]
Try `bot.dig(target, 'raycast')`
